gdx/test/com/badlogic/gdx/utils/QueueTest.java

```diff
@@ -1,16 +1,29 @@
-
 package com.badlogic.gdx.utils;
 
 import static org.junit.Assert.*;
 
 import java.util.Iterator;
+import java.util.NoSuchElementException;
 
 import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 public class QueueTest {
+
+	@Rule
+	public final ExpectedException exception = ExpectedException.none();
+
 	@Test
-	public void addFirstAndLastTest() {
+	public void initializeWithType () {
+		Queue<String> queue = new Queue<String>(1, String.class);
```

Comment:
with or without type initialization, this test should pass. Maybe you need to test direct access to underlying array instead.